rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -330,6 +330,28 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 
 	m_emu_settings->EnhanceRadioButton(ppu_bg, emu_settings_type::PPUDecoder);
 
+	connect(ppu_bg, &QButtonGroup::idToggled, [this](int id, bool checked)
```

Comment:
Ugh... Need to put this before EnhanceRadioButton

Reply:
Damn. Why though? It seems working.

Reply:
initial state